components/card/basic/src/index.js

```diff
@@ -84,7 +87,11 @@ CardBasic.propTypes = {
   /**
    * Lazy load flag / config.
    */
-  lazyLoad: PropTypes.oneOfType([PropTypes.bool, PropTypes.object])
+  lazyLoad: PropTypes.oneOfType([PropTypes.bool, PropTypes.object]),
+  /**
+   * Button for actions
+   */
+  button: PropTypes.object
```

Comment:
It would rather be a `PropTypes.node`, isn't it?

Reply:
Yes, that's true.